frontend-maven-plugin/src/it/custom-install-directory/verify.groovy

```diff
@@ -1,2 +1,2 @@
-assert new File(basedir, 'target/node').exists() : "Node was not installed in the custom install directory";
+assert new File(basedir, 'target/.buildenv/node').exists() : "Node was not installed in the custom install directory";
```

Comment:
I'm not sure it's such a good idea to add the .buildenv/ directory, since it will change the directory layout, which could hurt backwards compatibility. Maybe we can just have node/node(.exe), node/npm/xxx and node/yarn/xxx?